src/routes/ErrorPage.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "antd";
+import React, { ReactElement } from "react";
+import { ErrorResponse, Link, useRouteError } from "react-router-dom";
+
+import { FlexColumnAlignCenter } from "../styles/utils";
+
+const isErrorResponse = (error: unknown): error is ErrorResponse => {
+  return (error as ErrorResponse).status !== undefined && (error as ErrorResponse).statusText !== undefined;
+};
```

Comment:
@frasercl is there a better way for me to be checking this? `ErrorResponse` is a type so I can't check by instance.

Reply:
Could use `typeof (error as ErrorResponse).status === "number" && typeof (error as ErrorResponse).statusText === "string"` for a little bit of extra soundness, but yeah type guards for types like this are always gonna have this "weird trick" feel. For all TS's power to make static JS typechecking nice, it has no power to make runtime typechecking nice.

Reply:
Thanks!